backend/app/assets/javascripts/spree/backend/components/selectable_table.js

```diff
@@ -0,0 +1,16 @@
+Spree.ready(function() {
+  $('table.selectable-table').each(function() {
+    var selectableTableModel = new Backbone.Model({
+      allSelected: false,
+      selectedItems: []
+    })
+
+    new Spree.Views.Tables.SelectableTable({el: this, model: selectableTableModel});
+
+    if($(this).hasClass('return-items-table')) {
```

Comment:
Why hard-code this behavior here? Can we add a dedicated view model for the return items table instead?

Reply:
I made a dedicated view for return items table, let me know if it works for you.